headers/2017.4.17f1.h

```diff
@@ -1868,7 +1868,7 @@ struct Il2CppDecimal
         } v;
         uint64_t Lo64;
     } v;
-};
+} Il2CppClass;
```

Comment:
This looks like a bug to me - it should not be identifying `Il2CppDecimal` as `Il2CppClass`.